pkg/controllers/management/cluster/clustercontroller.go

```diff
@@ -74,7 +74,8 @@ func (c *controller) capsSync(key string, cluster *v3.Cluster) (runtime.Object,
 
 		kontainerDriver, err := c.kontainerDriverLister.Get("", driverName)
 		if err != nil {
-			return nil, fmt.Errorf("error getting kontainer driver: %v", err)
+			logrus.Warnf("Error getting kontainer driver: %v", err)
```

Comment:
Check the error to see what it is, if it's a not exist then it should be fine, otherwise give it back.

Reply:
But we need kontainerDrive.Name on line 83?

Reply:
You have to get out of this function if you have an error, check the type of error to determine what to do similar to the other file. 
```
if err != nil {
    !errors.IsNotFound(err) {
        wrap error
    }
    return obj, nil
}
````